Python/Product/Common/Strings.resx

```diff
@@ -495,6 +495,221 @@ The web.config file will be configured to resolve URIs matching this pattern to
   <data name="AddWebRoleSupportFiles" xml:space="preserve">
     <value>To ensure your project will deploy correctly to Microsoft Azure Cloud Service, add the Web Role Support Files item to your project through the Add New Item dialog and follow the instructions provided.</value>
   </data>
+  <data name="AddInterpreterWindow_AssociatedEnvironments" xml:space="preserve">
+    <value>Python environments associated with my project</value>
+  </data>
+  <data name="AddInterpreterWindow_AssociatedEnvironmentsHelp" xml:space="preserve">
+    <value>Select the environments to associate with this project. These environments will be shown in Solution Explorer.</value>
+  </data>
+  <data name="AddInterpreterWindow_HelpFind" xml:space="preserve">
+    <value>Go online and help me find another one</value>
+  </data>
+  <data name="AddInterpreterWindow_OK" xml:space="preserve">
+    <value>_OK</value>
+  </data>
+  <data name="AddInterpreterWindow_Cancel" xml:space="preserve">
+    <value>_Cancel</value>
+  </data>
+  <data name="AddInterpreterWindow_Title" xml:space="preserve">
+    <value>Add/Remove Python Environments</value>
+  </data>
+  <data name="AddVirtualEnvironmentWindow_BrowsePathError" xml:space="preserve">
+    <value>We could not identify a virtual environment in the selected path.</value>
+  </data>
+  <data name="AddVirtualEnvironmentWindow_BrowsePathOrigPrefixError" xml:space="preserve">
+    <value>We could not find a base interpreter at </value>
+    <comment>A folder path is concatenated to the end of this string</comment>
+  </data>
+  <data name="AddVirtualEnvironmentWindow_Actions" xml:space="preserve">
+    <value>Actions we will perform:</value>
+  </data>
+  <data name="AddVirtualEnvironmentWindow_InstallPipAndVirtualEnv" xml:space="preserve">
+    <value>Install pip and virtualenv packages</value>
```

Comment:
Just a note that we need to remember to specify that "pip" is a product name and should not be localized. I'm guessing they won't try to localize virtualenv.